packages/yew-router/src/history.rs

```diff
@@ -367,7 +367,7 @@ impl BrowserHistory {
         Self::default()
     }
 
-    fn route_to_url(route: impl Routable) -> Cow<'static, str> {
+    pub(crate) fn route_to_url(route: impl Routable) -> Cow<'static, str> {
```

Comment:
Why did you make this private, is there another way to get a url from a route?

Reply:
Well besides `route.to_path()` since that does not give you base

Reply:
If I understand correctly, you suggest to make this simply `pub`, in which case I'm not sure if `BrowserHistory` is actually the right place to find it for other people?

Reply:
Oh right it was private before my bad

Reply:
Leave it as is for now